.github/workflows/fossa.yml

```diff
@@ -0,0 +1,26 @@
+name: FOSSA
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
I think you need to add 

```yaml
container: kedacore/build-tools:latest
```

to use the environment built from https://github.com/kedacore/keda/blob/master/tools/build-tools.Dockerfile

Reply:
Ooops, thanks.